element_interface/prairieviewreader.py

```diff
@@ -5,107 +5,110 @@
 import numpy as np
 
 
-def get_pv_metadata(pvtiffile: str) -> dict:
-    """Extract metadata for scans generated by PrairieView acquisition software.
+def get_pv_metadata(ome_tif_filepath: str) -> dict:
```

Comment:
Will you also be updating the function name as below?

```suggestion
def get_prairieview_metadata(ome_tif_filepath: str) -> dict:
```

Reply:
I was debating this. I think since you've brought it up, I think it's a good idea. Committing.